src/Item/Item.router.ts

```diff
@@ -300,73 +308,82 @@ export class ItemRouter extends Router {
       )
     }
 
-    if (itemJSON.collection_id) {
-      const dbCollectionToAddItem = await Collection.findOne<CollectionAttributes>(
-        itemJSON.collection_id
+    const dbItem = await Item.findOne<ItemAttributes>(id)
+
+    const isItemCollectionBeingChanged =
+      itemJSON.collection_id && itemJSON.collection_id !== dbItem?.collection_id
+
+    if (isItemCollectionBeingChanged) {
+      throw new HTTPError(
+        "Item can't change between collections",
+        { id },
+        STATUS_CODES.unauthorized
       )
-      // So far, only the owner can add item if the collection was not published
-      if (
-        !dbCollectionToAddItem ||
-        dbCollectionToAddItem.eth_address.toLowerCase() !== eth_address
-      ) {
-        throw new HTTPError(
-          'Unauthorized user',
-          { id, eth_address, collection_id: itemJSON.collection_id },
-          STATUS_CODES.unauthorized
-        )
-      }
```

Comment:
Moved validation to [here](https://github.com/decentraland/builder-server/pull/270/files#diff-5db28707b1b7436bf2151ad65251263f4b79eeddc180a1a22996464abd5866c4R331)